parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -272,7 +273,7 @@ function configNetworkPolicy() {
 function installClearContainersRuntime() {
 	# Add Clear Containers repository key
 	echo "Adding Clear Containers repository key..."
-	curl -sSL "https://download.opensuse.org/repositories/home:clearcontainers:clear-containers-3/xUbuntu_16.04/Release.key" | apt-key add -
+	curl -sSL --retry 5 --retry-delay 10 --retry-max-time 30 "https://download.opensuse.org/repositories/home:clearcontainers:clear-containers-3/xUbuntu_16.04/Release.key" | apt-key add -
```

Comment:
not related to this PR, but are we ok to pull data from open internet? I kind of get the impression that we put a copy in Azure and download from there.

Reply:
@jessfraz might be able to answer, but this is out of scope for this PR since I'm only adding retries to all the curl commands.